lbrynet/daemon/Daemon.py

```diff
@@ -163,6 +163,12 @@ def __call__(self):
         return d
 
 
+def arrange_results(claims):
```

Comment:
Can you rename this to something more specific such as `sort_claim_results`? Just `arrange_results` doesn't give you enough information on what it's for.